.github/workflows/niv-updater.yml

```diff
@@ -1,32 +0,0 @@
-name: Update niv dependencies (daily)
```

Comment:
Wait a sec. There is a job that updates the `base` library in here. We shouldn't kill that! I thought you only wanted to remove the rare one and stop `ic-hs` from being built. (I think `ic-ref` is zapped since last week).

Reply:
Maybe go through closed PRs or updater branches and see which are still relevant.

Reply:
I see. Just read your note!